packages/flutter/lib/src/cupertino/switch.dart

```diff
@@ -428,6 +441,16 @@ class _RenderCupertinoSwitch extends RenderConstrainedBox {
     markNeedsPaint();
   }
 
+  Color get thumbColor => _thumbColor;
+  Color _thumbColor;
```

Comment:
nit: alternatively you can hold the `CupertinoThumbPainter` in an instance variable and `thumbColor` is just `thumbPainter.color`.

Reply:
I'm not sure that I comprehend your meaning ,so i add some codes below.

Reply:
Yeah that's what I meant.